resources/sip11/ra/src/main/java/org/mobicents/slee/resource/sip11/wrappers/DialogWrapper.java

```diff
@@ -501,9 +501,17 @@ public void delete() {
 		}
 		
 		final DialogState currentState = wrappedDialog != null ? wrappedDialog.getState() : null;
+		boolean isServer = wrappedDialog != null ? wrappedDialog.isServer() : false; // is false ok, here?
 		boolean stackDoesNotFiresDialogTerminatedEvent = !isEnding()
-			&& !wrappedDialog.isServer() && (currentState == null || currentState == DialogState.TERMINATED);
-		wrappedDialog.delete();
+			&& !isServer() && (currentState == null || currentState == DialogState.TERMINATED);
```

Comment:
@jaimecasero: Good catch! Indeed, this is my mistake and there supposed to be the local  variable "isServer" instead of "isServer()" call. 

I tested the change with isServer, and 3rd party tests were executed correctly. During PR preparation I written the code manually, instead of copy&paste. 

Will prepare an update.